srcStatic/GameModules/GameModule.h

```diff
@@ -11,10 +11,11 @@ class GameModule
 	inline std::string Name() const { return name; };
 
 	virtual void Load() = 0; //mod_init(), InitMod
-	// Only available for Console modules (other modules would just call an empty function)
-	//virtual void Run() = 0; //mod_run()
 	virtual bool Unload() = 0; //mod_destroy(), DestroyMod
 
+	// Only available for Console modules (other modules just call an empty function)
+	virtual void Run() { }; //mod_run()
```

Comment:
This documentation is probably more suited to the derived class. It's the derived class that potentially overrides this method and decides if it actually does anything. Adding support for this in INI modules would not require changes to the base class. It would only need to define a DLL method to call, and then start calling it from the derived class that handles INI modules.